war/src/main/webapp/images/jenkins-header-logo-new.svg

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
naming a file with `new` in it is generally a bad idea, it will likely stay forever.

possibly add `v2` ?

Reply:
Renamed to `jenkins-header-logo-v2.svg`. I have also renamed other references to the "new ui". For example, the new css files are `base-styles-v2.less` and `ui-refresh-overrides.less`. LESS variables were also changed.

Please reopen this convo if you feel the changes weren't right